lib/udap_security_test_kit/registration_success_contents_test.rb

```diff
@@ -15,14 +15,25 @@ class RegistrationSuccessContentsTest < Inferno::Test
       > use by the Client App, the software statement as submitted by the Client App, and all of the registration
       > related parameters that were included in the software statement.
 
+      [UDAP STU 1.1](https://hl7.org/fhir/us/udap-security/STU1.1/registration.html#request-body) clarifies that,
```

Comment:
This is a link to version 1.1 of the spec, but above we're linking to version 1.

Reply:
That was intentional, but maybe warrants more explanation? I did so based on [this Slack conversation](https://mitrecorp.slack.com/archives/C07RC2CQ34K/p1729184780869579) between you, me, and Rob, in which you stated that you considered that update to HL7 UDAP STU 1.1 a clarification that we could account for in our STU 1.0 tests. To explain our updated test behavior, I linked the STU 1.1 source but did not explain the thought process behind the change for conciseness.

I could either omit the STU 1.1 reference and just keep the RFC 7591 reference, or explain why STU 1.1 is included, i.e., "Even though these tests are for STU 1.0, Inferno interprets this specific update in STU 1.1 as a clarification and therefore does not require all field values returned by the server to exactly match those submitted in the registration request's software statement."